### ISSUE TITLE
Unexpected 'var_export' error

### ISSUE THREAD

**Issue by mcdemon** [spinen/connectwise-php-client#15]
Hi All,

I successfully installed the package and proceeded to the non-Laravel configuration step.  As I am updating the objects in the interactive PHP shell, I get to:

`var_export($info->toArray(), false);
array (
  'version' => 'v2018.5.56760',
  'isCloud' => false,
  'serverTimeZone' => 'Eastern Standard Time',
)`

then receive the following error. 

Parse error: syntax error, unexpected 'var_export' (T_STRING) in php shell code on line 6

![image](https://user-images.githubusercontent.com/6197486/45038821-0abedf00-b030-11e8-8089-633a260a227b.png)

I am dev'ing on a WAMP server with PHP version 7.1.9 and Apache 2.4.27

Any guidance is appreciated.

Thank You

**Comment by mcdemon** [comment]
Bump.  Anyone?

**Comment by mcdemon** [comment]
No one can offer guidance?

**Comment by ssfinney** [comment]
@mcdemon What version of PHP are you using? I'm testing with 7.1.21 at the moment and can't reproduce that particular error.

**Comment by mcdemon** [comment]
Hi @ssfinney .  I have tried the process with 5.6.31, 7.1.9 and 7.0.23 and received the error in each.  

Thanks in advance for the assistance.

**Comment by ssfinney** [comment]
It looks like you are still getting the expected output from the call, even though you're getting that syntax error.

1. Have you tried var_dump instead, just to check the difference?
2. Can you successfully access other methods on the `$info` results, like the following:

`var_export($info->toJson(), false);`
`var_export($info->isCloud, false);`

**Comment by mcdemon** [comment]
So I added 7.1.22 to my WAMP install and ran through the process again.  I now get the error below.  I do not get any errors when executing the two var_export commands you listed.  
![image](https://user-images.githubusercontent.com/6197486/45762532-8efe8e00-bbfc-11e8-80b3-478fd3985b45.png)

When I substitute 'dump' for 'export' in the command generating the error, I get an 'unexpected var_dump' error.

**Comment by ssfinney** [comment]
Please put semicolons at the end of your lines, so that your var_export can run and print what you need.

**Comment by mcdemon** [comment]
OK.  So I found my initial problem; I was not initiating the php interactive shell from the correct directory.  ..Once I opened it from the right folder and addid the ; back to the lines of code, I got past the initial error.  Now when I issue the var_export($info, false); command, the interactive shell hangs.


![image](https://user-images.githubusercontent.com/6197486/45767891-69c44c80-bc09-11e8-9c2e-26fbec5353f7.png)
